app/templates/content/index.hbs

```diff
@@ -79,7 +79,7 @@
                                 {{t "content.private_preprint_warning" documentType=model.provider.documentType supportEmail='support@osf.io'}}
                             </div>
                         {{/if}}
-                        {{preprint-file-renderer provider=model.provider preprint=model primaryFile=primaryFile dualTrackNonContributors=(action 'dualTrackNonContributors')}}
+                        {{preprint-file-renderer provider=model.provider preprint=model primaryFile=primaryFile trackNonContributors=(action 'trackNonContributors')}}
```

Comment:
best i can tell, `preprint-file-renderer` has never used `dualTrackNonContributors` (but has been invoked with it [ever since it was added](https://github.com/CenterForOpenScience/ember-osf-preprints/commit/570150f7e168429f38488de73bcacb298f29eef2)), so i think yes, it is vestigial:
```suggestion
                        {{preprint-file-renderer provider=model.provider preprint=model primaryFile=primaryFile}}
```